pkg/virt-handler/migration-proxy/migration-proxy.go

```diff
@@ -319,7 +331,7 @@ func (m *migrationProxy) createTcpListener() error {
 	var err error
 	if m.tlsConfig != nil {
 		listener, err = tls.Listen("tcp", fmt.Sprintf("%s:%d", m.tcpBindAddress, m.tcpBindPort), m.tlsConfig)
```

Comment:
Don't you need here [] around the tcpBindAddress in case of ipv6?

Reply:
ips when they were specific ip and not ::1 , already had '['
i can add print to make sure we pass here and see what the ip is

Reply:
• [SLOW TEST:137.866 seconds]
[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system] VM Live Migration
/root/go/src/kubevirt.io/kubevirt/tests/migration_test.go:60
  Starting a VirtualMachineInstance 
  /root/go/src/kubevirt.io/kubevirt/tests/migration_test.go:345
    migration security
    /root/go/src/kubevirt.io/kubevirt/tests/migration_test.go:869
      [test_id:2303][posneg:negative] should secure migrations with TLS
      /root/go/src/kubevirt.io/kubevirt/tests/migration_test.go:886


a TLS test passes, it means the ip already have it (and i did saw ips that have [ ] already)
else the listen wont success